nixos/modules/services/networking/consul.nix

```diff
@@ -61,31 +41,27 @@ in
         '';
       };
 
-      interface = {
-
-        advertise = mkOption {
-          type = types.nullOr types.str;
-          default = null;
-          description = ''
-            The name of the interface to pull the advertise_addr from.
-          '';
-        };
+      settings = mkOption {
+        type = format.type;
```

Comment:
```
undefined variable 'format' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-1/nixos/modules/services/networking/consul.nix:45:16
```

Reply:
Fixed. What are you running to detect this, and how can I run it?

Reply:
The ofborg checks failed and showed that.